### DISCUSSION TITLE
Addition of releases & release notes

### DISCUSSION THREAD

**Discussion by sassomedia** [LegendApp/legend-state#245] [Ideas]
Hi there, I love the concept of this library! Re-rendering in React drives me crazy. 😞 

I do have one suggestion though; it would be very helpful if you could publish releases and release notes of what got fixed, has been added, or breaking change notices. This would make dependency management of this library much easier.

Thank you! 🙏

**Comment by jmeistrich** [comment]
I try to keep the changelog up to date but I forget sometimes. I just updated that: https://github.com/LegendApp/legend-state/blob/main/CHANGELOG.md

I'm not sure how to automate creating releases and release notes. Is there a package that can do that automatically when releasing?

**Reply by sassomedia** [reply]
Right, manual updates can be tedious. The package [semantic-release](https://www.npmjs.com/package/semantic-release) automates this process as long as you follow commit message conventions. Those in turn could be enforced with [danger](https://www.npmjs.com/package/danger) or maybe some other package.
